shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/SurveyDetailApiModel.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.blisskmmic.data.model
+
+import co.nimblehq.blisskmmic.domain.model.SurveyDetail
+import co.nimblehq.blisskmmic.domain.model.Token
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SurveyDetailApiModel(
+    val title: String,
+    val description: String,
+    @SerialName("is_active") val isActive: Boolean,
```

Comment:
@blyscuit Can we have the linebreak here?
```suggestion
    @SerialName("is_active") 
    val isActive: Boolean,
```

Reply:
Please refer to this [discussion](https://github.com/blyscuit/kmm-swiftUI-IC/pull/106#discussion_r1065309629). We should derive some format for the team, but for this project we will keep this format. 🙏